src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/templates/letter/InternalDetGenerateHearingBundleTemplate.java

```diff
@@ -50,12 +51,13 @@ public Map<String, Object> mapFieldValues(
         fieldValues.putAll(getAppellantPersonalisation(asylumCase));
         fieldValues.put("dateLetterSent", formatDateForNotificationAttachmentDocument(LocalDate.now()));
 
-        final String listCaseHearingDate = asylumCase.read(LIST_CASE_HEARING_DATE, String.class)
-                .orElseThrow(() -> new RequiredFieldMissingException("List case hearing date not found."));
-        LocalDateTime hearingDateTime = LocalDateTime.parse(listCaseHearingDate);
-        fieldValues.put("hearingDate", formatDateForNotificationAttachmentDocument(hearingDateTime.toLocalDate()));
-        fieldValues.put("hearingTime", hearingDateTime.toLocalTime());
-        fieldValues.put("hearingLocation", hearingDetailsFinder.getHearingCentreName(asylumCase));
+        Optional<String> listCaseHearingDate = asylumCase.read(LIST_CASE_HEARING_DATE, String.class);
+        if (listCaseHearingDate.isPresent()) {
```

Comment:
I don't think this is correct, AFAIK list_case_hearing_date should be a mandatory field

Reply:
I have updated with empty strings